packages/purpose-process/.env

```diff
@@ -22,6 +22,8 @@ S3_BUCKET=interop-local-bucket
 S3_CUSTOM_SERVER=true
 S3_SERVER_HOST=http://localhost
 S3_SERVER_PORT=9000
+MINIO_ACCESS_KEY=testawskey
+MINIO_SECRET_KEY=testawssecret
```

Comment:
Leftover? We don't set these variables anywhere

Reply:
I think it's part form the .env refactor in every process done by @Viktor-K

Reply:
👌  so I'm pretty sure you can safely remove these vars, they're probably leftovers from a rebase, we don't have vars called like that in any other service or env file